pkg/sfu/buffer/rtpstats.go

```diff
@@ -279,6 +279,15 @@ func (r *RTPStats) Update(rtph *rtp.Header, payloadSize int, paddingSize int, pa
 	return
 }
 
+func (r *RTPStats) ForceUpdateLastPacket(rtph *rtp.Header, packetTime int64) {
+	r.lock.Lock()
+	defer r.lock.Unlock()
+
+	r.highestSN = rtph.SequenceNumber - 1
+	r.highestTS = rtph.Timestamp
+	r.highestTime = packetTime
+}
```

Comment:
mute/unmute, we are (or should be) re-syncing and the sequence number should not jump. If it is, it is a bug. I don't think we should force a change into stats.

Are you seeing this on up stream side (clients should be sending consecutive) or down stream side (we should be re-syncing)?